components/form/signUpForm.tsx

```diff
@@ -0,0 +1,181 @@
+              className="text-lg md:text-xl self-center"
+            >
+              {data.common.contactUsFormFirstName}
+            </label>
+            {formik.errors.firstName ? (
+              <div className="">{formik.errors.firstName}</div>
+            ) : null}
+            <input
+              className="w-11/12 h-10 md:h-14 border rounded-lg border-solid border-[#FFFFFF] box-border shadow-[0px_0px_4px_rgba(0,0,0,0.25)]"
+              id="firstName"
+              name="firstName"
+              type="text"
+              onChange={formik.handleChange}
+              value={formik.values.firstName}
+            />
+          </div>
+          <div className="grid grid-rows-2 ">
+            <label
+              htmlFor="lastName"
+              className="place-self-start text-lg md:text-xl pr-7 self-center"
+            >
+              {data.common.contactUsFormLastName}
+            </label>
+            <input
+              className="w-11/12 h-10 md:h-14 border rounded-lg border-solid border-[#FFFFFF] box-border shadow-[0px_0px_4px_rgba(0,0,0,0.25)] place-self-end"
+              id="lastName"
+              name="lastName"
+              type="text"
+              onChange={formik.handleChange}
+              value={formik.values.lastName}
+            />
+          </div>
+          <div className="grid grid-rows-2 ">
+            <label
+              htmlFor="email"
+              className="place-self-start text-lg md:text-xl self-center"
+            >
+              {data.common.contactUsFormEmail}
+            </label>
+            <input
+              className="w-11/12 h-10 md:h-14 border rounded-lg border-solid border-[#FFFFFF] box-border shadow-[0px_0px_4px_rgba(0,0,0,0.25)] "
+              id="email"
+              name="email"
+              type="email"
+              onChange={formik.handleChange}
+              value={formik.values.email}
+            />
+          </div>
+
+          <div className="grid grid-rows-2 ">
+            <label
+              htmlFor="phone"
+              className="place-self-start text-lg md:text-xl pr-7 self-center"
+            >
+              {data.common.contactUsFormPhone}
+            </label>
+            <input
+              className="w-11/12 h-10 md:h-14 border rounded-lg border-solid border-[#FFFFFF] box-border shadow-[0px_0px_4px_rgba(0,0,0,0.25)] place-self-end"
+              id="phone"
+              name="phone"
+              type="tel"
+              onChange={formik.handleChange}
+              value={formik.values.phone}
+            />
+          </div>
+        </div>
+
+        <div className="grid pt-4">
+          <label
+            htmlFor="email"
+            className="place-self-start text-lg md:text-xl pb-4"
+          >
+            {data.common.contactUsFormMessage}
+          </label>
+          <input
+            className="h-24 md:h-36 border rounded-lg border-solid border-[#FFFFFF] box-border shadow-[0px_0px_4px_rgba(0,0,0,0.25)]"
```

Comment:
define this shadow in tailwind.config

Reply:
fixed - shadow defined in tailwind.config